helper/imexport.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * DokuWiki Plugin struct (Helper Component)
+ *
+ * @license GPL 2 http://www.gnu.org/licenses/gpl-2.0.html
+ * @author  Andreas Gohr, Michael Große <dokuwiki@cosmocode.de>
+ */
+
+// must be run within Dokuwiki
+use dokuwiki\plugin\struct\meta\Schema;
+
+if(!defined('DOKU_INC')) die();
+
+class helper_plugin_struct_imexport extends DokuWiki_Plugin {
+
+    private $sqlite;
+
+
+    /**
+     * this possibly duplicates @see helper_plugin_struct::getSchema()
+     */
+    public function getAllSchemasList() {
+        return Schema::getAll();
+    }
+
+    /**
+     * Delete all existing assignment patterns of a schema and replace them with the provided ones.
+     *
+     * @param string   $schemaName
+     * @param string[] $patterns
+     */
+    public function replaceSchemaAssignmentPatterns($schemaName, $patterns) {
+        /** @var \helper_plugin_struct_db $helper */
+        $helper = plugin_load('helper', 'struct_db');
+        $this->sqlite = $helper->getDB();
+        $schemaName = $this->sqlite->escape_string($schemaName);
```

Comment:
I think it would make sense to extend the `doTransaction()` method of the sqlite-plugin to be able to handle sqlite-statements with extra value-arrays as well.